### DISCUSSION TITLE
Mob timer correction thread

### DISCUSSION THREAD

**Discussion by TheAngryRogue** [smasherprog/EqTool#74] [General]
In an attempt to consolidate the requests for mob kill timer corrections I am making this thread to help the developer. Please keep your posts in the method listed below to avoid clutter.
If the Custom Timer needs to be modified for a particular mob in a particular zone just add the them here in the following format: 

**Mob Name /  Mob Zone  / Timer in Minutes:Seconds format**

Example and first requested edit below.

**Comment by TheAngryRogue** [comment]
an Othmir Warrior / Cobalt Scar / 20:00

**Comment by smasherprog** [comment]
Above fixed. I had the wrong zone timer for everything in the zone. Fixed all of cs set to 20 min respawn time
